$NetBSD: patch-configure,v 1.1 2016/09/08 16:18:01 joerg Exp $

Bogus conditional is failing when using external Boost after the test
actually passed.

--- configure.orig	2016-08-27 21:03:55.775361937 +0000
+++ configure
@@ -10043,9 +10038,6 @@ fi
       if test "x$ax_lib" = "x"; then
         as_fn_error $? "Could not find a version of the library!" "$LINENO" 5
       fi
-      if test "x$link_date_time" != "xyes"; then
-        as_fn_error $? "Could not link against $ax_lib !" "$LINENO" 5
-      fi
     fi
 
     CPPFLAGS="$CPPFLAGS_SAVED"
